src/test/java/semantics/extension/RDFEndpointTest.java

```diff
@@ -1528,6 +1525,322 @@ public Void apply(GraphDatabaseService graphDatabaseService) throws RuntimeExcep
     }
   }
 
+  @Test
+  public void testCypherOnRDFDatasetSerializeAsTriG() throws Exception {
+    // Given
+    try (ServerControls server = getServerBuilder()
+        .withProcedure(RDFImport.class)
+        .withExtension("/rdf", RDFEndpoint.class)
+        .withFixture(new Function<GraphDatabaseService, Void>() {
+          @Override
+          public Void apply(GraphDatabaseService graphDatabaseService) throws RuntimeException {
+            try (Transaction tx = graphDatabaseService.beginTx()) {
+              graphDatabaseService.execute("CREATE INDEX ON :Resource(uri)");
+
+              tx.success();
+            } catch (Exception e) {
+              fail(e.getMessage());
```

Comment:
The test fails if a non-expected exception is thrown, so this try-catch block is not only not necessary, but also (imo) counter-productive, as the stack-trace will not be shown in the error logs (leading to more difficult debugging if something goes wrong).

I would just let the exception be thrown.

Reply:
Alternatively, in case this is problematic (since the signature defines `RuntimeException` and not `Exception`, which would mean having to wrap the exceptions) I would at least execute `e.printStackTrace()` before the `fail` command to avoid information loss.

The same should apply to all other similar cases in the test class.